app/service/HeatMapSourceGenerator.js

```diff
@@ -135,8 +173,7 @@ angular
             var params = {
                 "q.text": keyword,
                 "q.time": '[' + this.getFormattedDateString(reqParamsUi.minDate) + ' TO ' + this.getFormattedDateString(reqParamsUi.maxDate) + ']',
-                "q.geo": '[' + bounds.minX + ',' + bounds.minY + ' TO ' + bounds.maxX + ',' + bounds.maxY + ']',
-                "a.hm.limit": 1000
+                "q.geo": '[' + bounds.minX + ',' + bounds.minY + ' TO ' + bounds.maxX + ',' + bounds.maxY + ']'
```

Comment:
Oh, you probably mean the next line and the parameter `a.hm.limit` :smile: 

If so, it was moved to the config to make and will be added [somewhere else](https://github.com/annarieger/SolrHeatmap/blob/7302bb3b8837bec7c1ccbc03a26626b1b8a5afc5/app/service/HeatMapSourceGenerator.js#L94) since the method `getTweetsSearchQueryParameters` will be used for both of search and csv export.